hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/OzoneKeyLocation.java

```diff
@@ -79,4 +83,9 @@ public long getOffset() {
     return offset;
   }
 
+  /**
+   * Returns the KeyOffset of this Key.
+   */
+  public long getKeyOffset() { return KeyOffset; }
```

Comment:
```suggestion
  public long getKeyOffset() { 
    return keyOffset;
  }
```
blocks should have line breaks.